agrirouter-sdk-dotnet-standard-impl/Service/Common/X509CertificateService.cs

```diff
@@ -72,6 +73,23 @@ public static X509Certificate GetCertificate(OnboardResponse onboardResponse)
                 $"Could not create a certificate for the type '${onboardResponse.Authentication.Type}'");
         }
 
+        /// <summary>
+        /// Create a certificate for the given RouterDevice
+        /// </summary>
+        /// <param name="routerDevice">-</param>
+        /// <returns>A X509 certificate to use for the communication between endpoint and AR.</returns>
+        /// <exception cref="CouldNotCreateCertificateForTypeException">-</exception        
+        public static X509Certificate GetCertificate(RouterDevice routerDevice)
+        {
+            var onboardResponse = new OnboardResponse()
+            {
+                ConnectionCriteria = new Api.Dto.Onboard.Inner.ConnectionCriteria(),
+                Authentication = new Api.Dto.Onboard.Inner.Authentication()
+            };
+            onboardResponse.ConnectionCriteria.GatewayId = GatewayTypeDefinition.Mqtt;//RouterDevices only work with MQTT
```

Comment:
```suggestion
            //Since router devices only work with MQTT, we are setting a fixed value at this point.
            onboardResponse.ConnectionCriteria.GatewayId = GatewayTypeDefinition.Mqtt;
```
No inline comments, please. I would suggest a whole sentence as comment to make the purpose clearer.